src/main/java/duke/command/DefaultCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.Ui;
+import duke.Storage;
+import duke.task.*;
```

Comment:
Is there a need for this import statement as there are no classes from the task package used?